Descent3/init.cpp

```diff
@@ -986,6 +988,17 @@
 #include "debuggraph.h"
 #include "vibeinterface.h"
 
+/*
+#ifdef _WIN32
+#include <windows.h>
+#include <shlwapi.h>
+#pragma comment(lib, "shlwapi.lib")
+#else
+#include <limits.h>
+#include <unistd.h>
+#endif
+*/
```

Comment:
Old code from before implementing winterhearts suggestion. Commented in case it needed to be reverted, but forgot to remove after successful build and test.